src/redux/selectors/index.js

```diff
@@ -1,6 +1,34 @@
-export const getSearchDetails = state => state.imagesGalleryPage
+import { createSelector } from 'reselect'
+
+import { ITEMS_PER_PAGES } from 'common/constants'
+
+// General
+export const getIsLoading = state => state.imgDailyPage.isLoading
+
+//  Image gallery
+export const getImages = state => state.imagesGalleryPage.items
+
+export const getImagesSelector = createSelector(
+  getImages,
+  items => items.flatMap(({ links }) => links),
+)
 
+// Daily image
+export const getDailyImage = state => state.imgDailyPage.image
+
+// Search
+export const getSearchDetails = state => state.imagesGalleryPage
 export const getSearchText = state => state.imagesGalleryPage.searchText
+
+// Pagination
 export const getCurrentPage = state => state.imagesGalleryPage.pageIndex
+export const getTotalItemsOnPage = state => state.imagesGalleryPage.totalItems
+
+export const getTotalItemsOnPageSelector = createSelector(
+  getTotalItemsOnPage,
+  items => items.total_hits / ITEMS_PER_PAGES,
+)
```

Comment:
you should also try to minimise your selectors as much as possible.
For example this one does 2 things:
 1. it gets you the total items -> then it retrieves the **total_hits** from it
 2. divides the total hits into a number

you should have 2 selectors for each of the cases, like that:
```
export const getTotalItemsHitsSelector = createSelector(
  getTotalItemsOnPage,
  items => items.total_hits,
)

export const getTotalItemsOnPageSelector = createSelector(
  getTotalItemsHitsSelector,
  total_hits  => total_hits / ITEMS_PER_PAGES,
)
```